trl/data_utils.py

```diff
@@ -155,6 +204,7 @@ def maybe_apply_chat_template(
                 - Preference dataset: `"prompt"`, `"chosen"`, and `"rejected"`.
                 - Preference dataset with implicit prompt: `"chosen"` and `"rejected"`.
                 - Unpaired preference dataset: `"prompt"`, `"completion"`, and `"label"`.
+                - [Name to find]: `"prompt"`, `"completion"`, and `"labels"`.
```

Comment:
What do you think of `Stepwise preference dataset`?

Reply:
stepwise: ok
preference: not really, as preference means that you've chosen and rejected. Here it's more "unpaired preference". 
stepwise unpaired preference dataset?

Reply:
Yes, good idea

Reply:
As mentioned above, I am not sure if "stepwise unpaired preference" is confusing since (a) process supervision isn't the same as preference learning and (b) there exists multi-step preference learning and people (like me!) might conflate the two